### ISSUE TITLE
New version of morphology

### ISSUE THREAD

**Issue by alisafaya** [ahmetaa/zemberek-nlp#136]
After updating to 0.11.1 version of zemberek ,I faced some parsing problems like below :
```
TurkishMorphology morphology = TurkishMorphology.createWithDefaults();
Z3MarkovModelDisambiguator disambiguator = new Z3MarkovModelDisambiguator();
TurkishSentenceAnalyzer analyzer = new TurkishSentenceAnalyzer(morphology, disambiguator);


String sentence = "Neredesiniz ?, kimle geliyorsun";
SentenceAnalysis analysis = analyzer.analyze(sentence);

writeParseResult(analysis);
```

which gives me this results :
```
Word = Neredesiniz
[(UNK:neredesiniz) (Unk,Unk;Unkown)]
Word = ?
[(?:?) (Punc)]
Word = ,
[(,:,) (Punc)]
Word = kimle
[(UNK:kimle) (Unk,Unk;Unkown)]
Word = geliyorsun
[(gelmek:gel) (Verb;Pos+Prog:iyor+A2sg:sun)]
```

**Comment by alisafaya** [comment]
it cannot parse a lot of words like :
- kimsiniz
- neredesin
- nerelisiniz
... in version 0.10.0 there was no problem but in order to use SpellChecker i had to upgrade to this version .

**Comment by ahmetaa** [comment]
Thanks for the report. I hope to fix this in the next version. Probably in 3-4 weeks.

**Comment by ahmetaa** [comment]
We are rewriting the morphology and disambiguation modules. Hopefully these problems will be fixed.

**Comment by ahmetaa** [comment]
These are fixed in 0.12.0

neredesin
[nere:Pron,Ques] nere:Pron+A3sg+de:Loc|Zero→Verb+Pres+sin:A2sg
kimsiniz
[kim:Pron,Ques] kim:Pron+A3sg|Zero→Verb+Pres+siniz:A2pl
nerelisiniz
[nere:Pron,Ques] nere:Pron+A3sg|li:With→Adj|Zero→Verb+Pres+siniz:A2pl
